CombatTagPlus/src/main/java/net/minelink/ctplus/event/PlayerCombatUntagEvent.java

```diff
@@ -0,0 +1,30 @@
+package net.minelink.ctplus.event;
+
+import java.util.UUID;
+
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+
+public final class PlayerCombatUntagEvent extends Event {
+    private UUID playerId;
```

Comment:
https://en.wikipedia.org/wiki/Law_of_triviality

Do you want me to resubmit with final added?